Autoprefixer.py

```diff
@@ -48,8 +48,19 @@ def has_selection(self):
 				return True
 		return False
 
-	def get_setting(self, key):
-		settings = self.view.settings().get('Autoprefixer')
-		if settings is None:
-			settings = sublime.load_settings('Autoprefixer.sublime-settings')
-		return settings.get(key)
+
+class AutoprefixerPreSaveCommand(sublime_plugin.EventListener):
+	def on_pre_save(self, view):
+
+		if get_setting(view, 'prefixOnSave') is True and is_css(view):
+				view.run_command('autoprefixer')
+
+
+def get_setting(view, key):
+	settings = view.settings().get('Autoprefixer')
+	if settings is None:
+		settings = sublime.load_settings('Autoprefixer.sublime-settings')
+	return settings.get(key)
+
+def is_css(view):
+		return view.settings().get('syntax') == 'Packages/CSS/CSS.tmLanguage'
```

Comment:
Actually, in latest dev version of Sublime, it's `Packages/CSS/CSS.sublime-syntax`. So I guess we should only check the file name (not path) without extension.